### DISCUSSION TITLE
issue with files

### DISCUSSION THREAD

**Discussion by gerlads** [KSP-CKAN/CKAN#3823] [Q&A] [answered]
from what i know i need to get CKAN to recognize the build.txt file in the ksp folder,but CKAN isnt seeing any files otehr than file folders,my KSP and CKAN are both on the same disk,but CKAN still doesnt see the file,im using the most recent version of ckan btw

**Comment by HebaruSan** [comment, marked as answer]
Hi, yes, when you add a game folder, you should select the buildID.txt file for KSP1:

https://github.com/KSP-CKAN/CKAN/wiki/User-guide#adding-game-folders

For KSP2, you select the KSP2_x64.exe file instead. The documentation isn't updated for this yet.

**Reply by HebaruSan** [reply]
Thanks for the heads-up, @gerlads, the user guide is now updated with info about support for KSP2.
